src/vue/common/WarningBanner.vue

```diff
@@ -19,19 +22,27 @@ export default {
 
 .warning-banner {
   min-height: 8rem;
-  padding: 1.6rem 4.8rem;
-  background-color: $col-warning;
+  padding: 1.2rem 4.8rem;
   font-size: 1.6rem;
   color: #fff;
   display: flex;
   align-items: center;
   justify-content: center;
+
+  &--warning {
+    background-color: $col-warning;
+  }
+
+  &--danger {
+    background-color: $col-danger;
+  }
 }
 
 .warning-banner__message-link {
   color: #fff;
   cursor: pointer;
   text-decoration: underline;
+  text-align: center;
```

Comment:
`.warning-banner__message-link` is unused isnt it?

Reply:
yap, removed